### PR TITLE
New unit types: force, power, and pressure

### PR DESCRIPTION
**Pull request by ehoogerbeets**
### Checklist

* [x] At least one test case is included for this feature or bug fix.
* [x] `ReleaseNotes` has added or is not needed.
* [x] This PR has passed all of the test cases on `Nodejs` and `Chrome Browser`.
* [x] This PR has passed all of the test cases on `QT/QML`.
* [ ] This is an API breaking change.
* [ ] Requires a major version change.

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

File: js/lib/PowerUnit.js [hunk 1]
```diff
@@ -0,0 +1,246 @@
+/*
+ * Power.js - Unit conversions for Power
+ *
+ * Copyright © 2018-2022, JEDLSoft
```
[the remark this answers is not in the document]
Comment (ehoogerbeets) [reply js/lib/PowerUnit.js:4]: Yes, good point. I did most of the work in 2021 and never updated them.

File: js/test/units/testPressure.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        });
+        var m2 = m1.scale("metric");
+
+        test.equal(m2.amount, 10);
+        test.equal(m2.unit, "millipascal");
+        test.done();
+    },
+
+/*
+    testPressureScale3: function(test) {
```
[the remark this answers is not in the document]
Comment (ehoogerbeets) [reply js/test/units/testPressure.js:131]: Originally I copied testPower.js to testPressure.js and then updated the tests to test pressure units. But, these two were unused. I'll remove them.

**Comment by ehoogerbeets** [comment]
Okay ready for re-review

File: js/lib/ForceUnit.js [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+    ],
+    "imperial": [
+        "poundal",
+        "pound-force"
+    ],
+    "conversions": {
+        "metric": {
+            "uscustomary": {
+                "millinewton": "pound-force",
+                "newton": "pound-force",
```
Comment (gouniLee) [inline js/lib/ForceUnit.js:113]: Question) No mapping for poundal?

File: js/lib/ForceUnit.js [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+        return undefined;
+    }
+    return force * fromRow[toRow[0]];
+};
+
+/**
+ * @private
+ * @static
+ */
+ForceUnit.getMeasures = function () {
```
Comment (gouniLee) [inline js/lib/ForceUnit.js:211]: just thought: We have two very similar name functions `getMeasure` and `getMeasures` Wouldn't it confuse users?

File: js/lib/ForceUnit.js (hunk 3 above)
Comment (ehoogerbeets) [reply js/lib/ForceUnit.js:113]: The poundal is not used very often to measure any common things.

File: js/lib/ForceUnit.js (hunk 4 above)
Comment (ehoogerbeets) [reply js/lib/ForceUnit.js:211]: That is how the design is for all of the units. We are sort of stuck with it now. We can deprecate `getMeasures` and make it an alias for `getMeasuresList` I guess.

Comment (gouniLee) [reply js/lib/ForceUnit.js:211]: Yeah, I checked all other unit features have the same functions. I agree to make an alias and deprecated getMearues

**Review by gouniLee — APPROVED** [review]

**Pull request merged**